scrc/enums/split.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class Split(Enum):
+    ALL = "all"
+    VAL = "val"
+    TEST = "test"
+    TRAIN = "train"
+    SECRET_TEST = "secret_test"
```

Comment:
Thanks. Did you check if this is used in all places where previously the strings were used?

Reply:
Great, thanks.
Please use the enum consistently everywhere. This is a good example of a benefit of the enum (the same name is used everywhere)

Reply:
Great, thanks.
Please use the enum consistently everywhere. This is a good example of a benefit of the enum (the same name is used everywhere)

Reply:
Alright